streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorTopology.java

```diff
@@ -197,4 +197,14 @@ public String toString(final String indent) {
         return sb.toString();
     }
 
+    // for testing only
+    public Set<String> processorConnectedStateStores(final String processorName) {
```

Comment:
Hmm - is there another way we can test this? Don't really like having methods that are purely for testing

Reply:
I have thought about this, via `TopologyDescription` but the code path would be very awkward. I ended up adding this function mainly because `ProcessorTopology` is an internal class so I felt it could be fine.